src/acl/Acl.h

```diff
@@ -69,6 +69,7 @@ class ACL
     virtual void parse() = 0;
     virtual char const *typeString() const = 0;
     virtual bool isProxyAuth() const;
+    virtual bool isCaseInsensitive() const { return false; }
```

Comment:
It feels wrong to assign a case sensitivity property to all ACLs. ACLs like `maxconn` should not be forced to decide whether they are case sensitive (and using the default "no" answer is still such a decision, at least on conceptual level).

AFAICT, the ACL data that needs case-sensitivity information should extract it from its ACL object via ACL::options() instead.